### PR TITLE
feat: initial implementation

### PR DESCRIPTION
**Pull request by keithamus**
This implements a `position: sticky` style component which uses [Stickyfill](https://github.com/wilddeer/stickyfill). It works better than our current implementation.

Page over at https://keithamus.github.io/component-stickyfill/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    ]
+  },
+  "stylelint": {
+    "extends": [
+      "stylelint-config-strict"
+    ]
+  },
+  "dependencies": {
+    "react": "^0.14.7",
+    "react-dom": "^0.14.8",
```
Comment (danielmoll) [inline package.json]: Not a problem, just noticed there's react 0.14.7 and react-dom 0.14.8

**Comment by danielmoll** [comment]
LGTM 👍

**Comment by danielmoll** [comment]
Do you need a couple of people to check this over before merge?

**Comment by keithamus** [comment]
Perhaps @umbobabo and @gcedo could check this over, as @umbobabo has implemented third party sticky components before, and @gcedo is aware of the recent bugs we've been having.

**Comment by fabiosantoscode** [comment]
LGTM

**Comment by gcedo** [comment]
LGTM

**Comment by keithamus** [comment]
@umbobabo if you're happy can you press the big green button please!

**Pull request merged**